WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -208,6 +250,9 @@ class GutenbergViewController: UIViewController, PostEditor {
 
         gutenberg.delegate = self
         showInformativeDialogIfNecessary()
+        if isOpenedDirectlyForPhotoPost {
+            showMediaSelectionOnStart()
```

Comment:
Maybe we can call this after the first Gutenberg layout is completed?
Same way as we call focus on title. This would prevent the toolbar layout issue.

Reply:
Like this:
https://github.com/wordpress-mobile/WordPress-iOS/blob/26728aaed030d717c8e094c653ff9a9728f07c50/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift#L488

Reply:
Scenario 1: I agree it must be a bug on the JS side that will need to be addressed separately on another PR. Until now we didn't have a way to insert multiple images in one go so this was unnoticed.

Scenario 2: I followed your suggestion and I think it's now solved.